Command/InitCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Creates the folder for the zend lucene search index if it does not exist yet.
+ */
+class InitCommand extends ContainerAwareCommand
+{
+    public function configure()
+    {
+        $this->setName('massive:search:init');
+        $this->setDescription('Initializes the data directory for the search bundle');
```

Comment:
Add a functional test, it need only execute the command see `Tests\Functional\Command\` tests.